milvus/utils/Function.ts

```diff
@@ -75,3 +75,49 @@ export const buildDefaultSchema = (data: {
     },
   ];
 };
+
+function convertToCamelCase(str: string) {
+  return str.replace(/_(.)/g, function (match, letter) {
+    return letter.toUpperCase();
+  });
+}
+
+export const getDataKey = (type: DataType, camelCase: boolean = false) => {
+  let dataKey = '';
+  switch (type) {
+    case DataType.FloatVector:
+      dataKey = 'float_vector';
```

Comment:
we used hard code string like `float_vector ` in many places.
May be we can use enum for these data types in another pr.